controller/image_layout_test.go

```
@@ -0,0 +1,62 @@
+	if !strings.Contains(content, "-o bin/telemetry") {
+		t.Error("make build must produce bin/telemetry")
+	}
+	if !strings.Contains(content, "cmd/telemetry") {
+		t.Error("make build must compile cmd/telemetry")
+	}
+}
+
+func TestMakefileDockerBuildCIStagesTelemetry(t *testing.T) {
+	content := mustRead(t, "Makefile")
+	if !strings.Contains(content, "-o bin/ci-stage/controller/telemetry") {
+		t.Error("make docker-build-ci must stage telemetry next to manager and router so Containerfile.prebuilt COPY . . places /telemetry")
```

golangci-lint (2.13.2)

[error] The line is 134 characters long, which exceeds the maximum of 120 characters.

(lll)